device-types/Cisco/C1000-16P-E-2G-L.yaml

```diff
@@ -0,0 +1,88 @@
+---
+manufacturer: Cisco
+model: C1000-16P-E-2G-L
+slug: c1000-16p-e-2g-l
+part_number: C1000-16P-E-2G-L
+comments: '[Cisco Catalyst 1000 Series Switches Data Sheet](https://www.cisco.com/c/en/us/products/collateral/switches/catalyst-1000-series-switches/nb-06-cat1k-ser-switch-ds-cte-en.html)'
+u_height: 1
+is_full_depth: false
+console-ports:
+  - name: con0
+    type: rj-45
+  - name: USB con0
+    type: usb-mini-b
+power-ports:
+  - name: External PSU
+    type: dc-molex-4plug
```

Comment:
This is not a supported type at the moment. You can use `dc-terminal` or `other`, or any other that is correct and supported.